torch/testing/_asserts.py

```diff
@@ -708,23 +706,27 @@ def assert_close(
     check_stride: bool = True,
     msg: Optional[Union[str, Callable[[Tensor, Tensor, SimpleNamespace], str]]] = None,
```

Comment:
I like your thinking here. What if we just give it a more friendly user-facing name like "DiagnosticInfo"?